File: p002/002.js
let a = 0,
  b = 1,
  result;

result = b;
let counter = 1;

for (let i = 1; i <= 50; i++) {
  console.log(`${counter}. ${result}`);
  result = a + b;
  a = b;
  b = result;
  counter++;
}

File: p002/002.py
a, b = 0, 1

result = b
counter = 1
sumOfNums = 0
limit = 4000000

for x in range(limit):
    if(result % 2 == 0 and result <= limit):
        sumOfNums += result
        result = a+b
        a = b
        b = result
        counter += 1
    else:
        result = a+b
        a = b
        b = result

print("The sum is", str(sumOfNums))
